deploy_docs.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
how is this licensed?

Reply:
This script comes from a code with the BSD license here: https://github.com/AMReX-Codes/amrex/blob/master/LICENSE. 

My experience of software licensing is limited, but from what I can gather from the internet, it would be sufficient to add this to the end of the existing license with a note along the lines of `deploy_docs.sh was modified from the original script in the AMReX code (https://github.com/AMReX-Codes/amrex), which is covered by the following license`